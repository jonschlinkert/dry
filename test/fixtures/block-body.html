{% extends "block-body" %}

{% block "body" %}
âˆš body: block-body.html
{% endblock %}
