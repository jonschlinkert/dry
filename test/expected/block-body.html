
√ header: default

√ body: block-body.html

√ footer: default
